packages/ui/src/components/Table/Row/styles.css.ts

```diff
@@ -31,10 +31,6 @@ export const row = style({
 		[`${tableStyles.table}:not(${tableStyles.noBorder}) ${headStyles.head} &:first-of-type`]:
 			{
 				borderTop: BORDER,
-			},
-
-		[`${tableStyles.table}:not(${tableStyles.noBorder}) ${headStyles.head} &:first-of-type`]:
```

Comment:
Fixes issue where table is missing top boarder

![Screenshot 2023-09-18 at 3 15 35 PM](https://github.com/highlight/highlight/assets/17744174/e6f07213-ff39-421a-8637-a25150a85e51)